eng/Publishing.props

```diff
@@ -32,7 +32,8 @@
         Condition=" '$(PublishInstallerBaseVersion)' == 'true' " />
     <_InstallersToPublish Include="$(ArtifactsDir)installers\**\*.wixlib" UploadPathSegment="Runtime" />
     <_InstallersToPublish Include="$(ArtifactsDir)installers\**\*.zip" UploadPathSegment="Runtime" />
-    <_ChecksumsToPublish Include="$(ArtifactsDir)**\*.sha512" />
+    <_ChecksumsToPublish Include="$(ArtifactsDir)packages\**\*.sha512" />
```

Comment:
Can't use `$(ArtifactsDir)**\*.sha512` with source build as the inner clone gets searched and picks up the checksums of some packages that get restored.

Reply:
FYI @dotnet/aspnet-build? I assumed it's verified all our *.sha512 files are from installers and packages in both public and internal builds?

Reply:
Upon taking a second look.  It not necessary to scan packages because we only generate checksums for items in installers - https://github.com/dotnet/aspnetcore/blob/main/eng/AfterSigning.targets#L21

Reply:
Yep, just installers